src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -261,10 +261,10 @@ def user_and_role(user, role = nil, options = {})
       icon = user_icon(user)
     end
 
-    if !(real_name.empty? || opt[:short])
-      printed_name = "#{real_name} (#{user})"
-    else
+    if real_name.empty? || opt[:short]
```

Comment:
This is not a test, this is changing the code. Why? and why is in the test commit? :thinking: